extensionsGallery.json

```diff
@@ -5080,7 +5080,68 @@
 					],
 					"statistics": [],
 					"flags": ""
-				}
+				},
+				{
+				"extensionId": "100",
```

Comment:
please also bump up the total count at the end of this file.

Reply:
Updated